docs/src/main/paradox/ibm-eventstore.md

```diff
@@ -0,0 +1,86 @@
+# IBM EventStore
+
+IBM EventStore is an in-memory database built on top of Apache Spark and Apache Parquet Data Format, the Alpakka IBM EventStore connector provides an Akka Stream flow and sink for insertion of records into a table.
+ 
+## External references 
+
+[IBM EventStore Documentation](https://www.ibm.com/support/knowledgecenter/SSGNPV/eventstore/desktop/welcome.html)
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-ibm-eventstore" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.lightbend.akka</groupId>
+      <artifactId>akka-stream-alpakka-ibm-eventstore_$scalaBinaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.lightbend.akka", name: "akka-stream-alpakka-ibm-eventstore_$scalaBinaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
```

Comment:
We have a new nifty paradox tag/fragment for these now, look at the other connector docs for a sample.